modules/generator/processor/localblocks/query_range.go

```diff
@@ -50,12 +50,17 @@ func (p *Processor) QueryRange(ctx context.Context, req *tempopb.QueryRangeReque
 		concurrency = uint(v)
 	}
 
+	exemplarsEnabled := req.Exemplars
```

Comment:
I think we can centralize the hint-checking logic into the CompileMetricsQueryRange/MetricsEvaluator.   It looks like the local-blocks processor isn't using the exemplars bool directly, just passing it around.  The Compile/Evaluator will already have access to the request, query, and unsafe setting, so they have everything needed.

Reply:
Good point, done.